projects/firefox/build.sh

```diff
@@ -45,6 +45,13 @@ export MOZCONFIG=$SRC/mozconfig.$SANITIZER
 export SHELL=/bin/bash
 ./mach bootstrap --no-interactive --application-choice browser
 
+# Skip patches for now
+rm tools/fuzzing/libfuzzer/patches/*.patch
+touch tools/fuzzing/libfuzzer/patches/dummy.patch
+
+# The git version used on oss-fuzz does not support --shallow-since
```

Comment:
This is not correct, we do support latest. So, i am removing it. Please update image using 
python infra/helper.py pull_images.
Images now have rust nightly, latest python, latest git, etc.